collections/_posts/2022-06-16-Recursive-Refinement-Network.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: review
+title: Recursive refinement network for deformable lung registration
+tags: deep-learning CNN registration medical lung CT
+cite:
+    authors: "He X, Guo J, Zhang X, Bi H, Gerard S, Kaczka D, Motahari A, Hoffman E, Reinhardt J, Barr RG, Angelini E."
+    title:   "Recursive Refinement Network for Deformable Lung Registration between Exhale and Inhale CT Scans"
+    venue:   "arXiv preprint arXiv:2106.07608. 2021 Jun 14"
+pdf: "https://arxiv.org/pdf/2106.07608"
+---
+
+# Introduction
+
+In this paper a recursive refinement network (RRN) is proposed for end-to-end unsupervised deformable registration. The performance of the algorithm is tested on [DirLab COPDGene](https://med.emory.edu/departments/radiation-oncology/research-laboratories/deformable-image-registration/index.html) dataset including 10 inhale/exhale lung CT images of COPD patients with almost 300 landmarks on each image. The network achieved a state-of-the-art average target registration error (TRE) of 0.83 mm.
+
+# Methods
+1. Multi-level features are extracted from fixed and moving images through 3D convolutional layers. 
+2. Features are normalized (to avoid feature vanishing at higher levels, as intermediate deformation vector fields are not supervised).
+3. The moving features are warped with the 2x upsampled DVF predicted at previous level (no warping for the topmost level).
```

Comment:
Precise that DVF means Deformable (or Displacement) Vector Field (at least I guess ;))